File: salary.py
#Program to calculate the salary of an employee
basicpay = float(input("Enter the basic pay of the employee: "))
float;ta = 0.4*basicpay
float;da = 0.2*basicpay
float;hra = 0.1*basicpay
grosspay = basicpay + ta + da + hra
print("The gross pay of the employee is: ",grosspay)

File: salary.c
//Program to calculate total salary
#include<stdio.h>
int main()
{float basicpay,hra,ta,grosspay;
 printf("Enter basic pay:");
 scanf("%f",&basicpay);
 hra = 0.2 * basicpay;
 ta = 0.15 * basicpay;
 grosspay = hra + ta + basicpay;
 printf("Your Total Salary is %f",grosspay);
 return 0;
}   